services/cinema_hall.py

```diff
@@ -0,0 +1,13 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> None:
+    return CinemaHall.objects.create(name=hall_name,
+                                     rows=hall_rows,
+                                     seats_in_row=hall_seats_in_row)
```

Comment:
fix indentation
or as the optional solution, you can have this
```suggestion
def create_cinema_hall(*args, **kwargs) -> None:
    return CinemaHall.objects.create(*args,**kwargs)
```
Also, fix here type annotation. It should be something else not None

Reply:
what happen with indentation?